src/webportal/src/app/job-submission/components/form-text-field.jsx

```diff
@@ -30,8 +30,6 @@ import { BasicSection } from './basic-section';
 import { FormShortSection } from './form-page';
 import { DebouncedTextField } from './controls/debounced-text-field';
 
-const TEXT_FILED_REGX = /^[A-Za-z0-9\-._~]+$/;
```

Comment:
Why remove this line?

Reply:
Username and taskrole need to use different regulars, so use the `regx` parameter to pass it in.